Prace_domowe/Praca_domowa2/Grupa4/Mateusz Stączek/notebook.ipynb

```diff
@@ -0,0 +1,789 @@
+{
```

Comment:
<p>Cieszę się, że jesteś&nbsp;świadomy, ale to nie jest dobrze \:(</p><p> </p><p>Fajnie jeszcze pokazać jak wygląda rozkład cechy, żebyśmy wiedzieli do czego porównać błąd</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Teraz nie wiem, czy chodzi Ci o RMSE na całej kolumnie zamiast na imputacji tylko, czy na obu kolumnach jednocześnie, a nie pojedynczo. To pierwsze jest bardzo złe, to drugie mniej, ale warto zobaczyć dla każdej kolumny osobno</p>